### DISCUSSION TITLE
Language label on account

### DISCUSSION THREAD

**Discussion by Rystars** [ppy/osu#16786] [Q&A] [answered]
![IMG_20220205_175617](https://user-images.githubusercontent.com/96302690/152639484-e905f518-02c0-4d51-9cb8-e4775c2b12ba.jpg)

Please make it able to change language

**Comment by bdach** [comment, marked as answer]
that is not language, that is the nationality of the user. it deliberately cannot be changed anymore. see notes from latest community meeting for reasoning: https://docs.google.com/document/d/1W_97ttbAo1mHjUgTeU_IB5SQVeQztT-pRrwiyTfjTu4
